src/modules/components/member/memberAvatar/memberAvatar.test.tsx

```diff
@@ -0,0 +1,91 @@
+import { render, screen, waitFor } from '@testing-library/react';
+import { getAddress, isAddress } from 'viem';
+import { normalize } from 'viem/ens';
+import { useEnsAddress, useEnsAvatar, useEnsName } from 'wagmi';
+import { MemberAvatar, type IMemberAvatarProps } from './memberAvatar';
+
+jest.mock('viem', () => ({
+    isAddress: jest.fn(),
+    getAddress: jest.fn(),
+}));
+
+jest.mock('viem/ens', () => ({
+    normalize: jest.fn(),
+}));
+
+jest.mock('wagmi', () => ({
+    useEnsAddress: jest.fn(),
+    useEnsName: jest.fn(),
+    useEnsAvatar: jest.fn(),
+}));
+
+describe('<MemberAvatar /> component with props', () => {
+    const createTestComponent = (props?: Partial<IMemberAvatarProps>) => {
+        const completeProps: IMemberAvatarProps = { ...props };
+
+        return <MemberAvatar {...completeProps} />;
+    };
+
+    const originalGlobalImage = global.Image;
+
+    beforeAll(() => {
+        (window.Image as unknown) = class MockImage {
+            onload: () => void = () => {};
+            src: string = '';
+            constructor() {
+                setTimeout(() => {
+                    this.onload();
+                }, 100);
+            }
+        };
+    });
+
+    afterAll(() => {
+        global.Image = originalGlobalImage;
+    });
+
+    beforeEach(() => {
+        jest.clearAllMocks();
+
+        (isAddress as unknown as jest.Mock).mockImplementation(() => true);
+        (getAddress as unknown as jest.Mock).mockImplementation(() => '0x028F5Ca0b3A3A14e44AB8af660B53D1e428457e7');
+        (useEnsAddress as jest.Mock).mockReturnValue({ data: null, isLoading: false });
+        (useEnsName as jest.Mock).mockReturnValue({ data: null, isLoading: false });
+        (useEnsAvatar as jest.Mock).mockReturnValue({ data: null, isLoading: false });
+    });
+
+    it('should display the avatar directly from avatarSrc prop and ensure no data fetching is attempted', async () => {
```

Comment:
We don't usually add the `should` on other tests, just to align the naming with the other components. Please remove it on the tests below as well
```suggestion
    it('displays the avatar directly from avatarSrc prop and ensure no data fetching is attempted', async () => {
```